src/foam/nanos/logger/ILogger.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.logger;
+
+public interface ILogger {
```

Comment:
the name Logger is already used by java, so i made it ILogger to signify its an (I)nterface

I can change it